addons/crm_helpdesk/crm_helpdesk.py

```diff
@@ -36,92 +14,90 @@ class crm_helpdesk(osv.osv):
-            'email_from': fields.char('Email', size=128, help="Destination email for email gateway"),
-            'date': fields.datetime('Date'),
-            'ref': fields.reference('Reference', selection=openerp.addons.base.res.res_request.referencable_models),
-            'ref2': fields.reference('Reference 2', selection=openerp.addons.base.res.res_request.referencable_models),
-            'channel_id': fields.many2one('utm.medium', 'Channel', help="Communication channel."),
-            'planned_revenue': fields.float('Planned Revenue'),
-            'planned_cost': fields.float('Planned Costs'),
-            'priority': fields.selection([('0','Low'), ('1','Normal'), ('2','High')], 'Priority'),
-            'probability': fields.float('Probability (%)'),
-            'categ_id': fields.many2one('crm.helpdesk.category', 'Category'),
-            'duration': fields.float('Duration', states={'done': [('readonly', True)]}),
-            'state': fields.selection(
-                [('draft', 'New'),
-                 ('open', 'In Progress'),
-                 ('pending', 'Pending'),
-                 ('done', 'Closed'),
-                 ('cancel', 'Cancelled')], 'Status', readonly=True, track_visibility='onchange',
-                                  help='The status is set to \'Draft\', when a case is created.\
-                                  \nIf the case is in progress the status is set to \'Open\'.\
-                                  \nWhen the case is over, the status is set to \'Done\'.\
-                                  \nIf the case needs to be reviewed then the status is set to \'Pending\'.'),
-    }
+    id = fields.Integer(string='ID', readonly=True)
+    name = fields.Char(string='Name', required=True)
+    active = fields.Boolean(string='Active', required=False, default=True)
+    date_action_last = fields.Datetime(string='Last Action', readonly=True)
+    date_action_next = fields.Datetime(string='Next Action', readonly=True)
+    description = fields.Text(string='Description')
+    create_date = fields.Datetime(string='Creation Date', readonly=True)
+    write_date = fields.Datetime(string='Update Date', readonly=True)
+    date_deadline = fields.Date(string='Deadline')
+    user_id = fields.Many2one('res.users', string='Responsible', default=lambda self: self.env.user)
+    team_id = fields.Many2one('crm.team', string='Sales Team', oldname='section_id',\
+                    select=True, help='Responsible sales team. Define Responsible user and Email account for mail gateway.')
+    company_id = fields.Many2one('res.company', string='Company', 
+                    default=lambda self: self.env['res.company']._company_default_get('crm.helpdesk'))
+    date_closed = fields.Datetime(string='Closed', readonly=True)
+    partner_id = fields.Many2one('res.partner', string='Partner')
+    email_cc = fields.Text(string='Watchers Emails', size=252,
```

Comment:
As per the task: 5400, we need to keep size attribute for email related fields.
task pad: https://pad.odoo.com/p/r.Vd4ZdqhkfgWN8cw9